### ISSUE TITLE
Running latest AAT policies is producing a global CSS bug

### ISSUE THREAD

**Issue by elizabethsjudd** [IBM/carbon-components#2049]
When running the latest IBMa AAT policies (`IBM_Accessibility_2019_02` and `IBM_Accessibility_WCAG21_2019_02`)

Violation:
```
"wcag21.tech.g1200.styleViewport": "Viewport units used to size text attributes do not rescale."
```

## Environment

> Operating system

OS X

> Browser

All

> Automated testing tool and ruleset

AAT

> Assistive technology used to verify

IBMa Karma Plugin

## Detailed description

> What version of the Carbon Design System are you using?

9.68.15

> What did you expect to happen? 

I should not be getting global violations when running AAT

> What happened instead?

Getting violation that text sizes are defined using viewport units

> What WCAG 2.1 checkpoint does the issue violate?

g1200

## Steps to reproduce the issue

Run `@ibma/karma-ibma` or internal DAP extension  using the above policies on a page with carbon css loaded.
